src/platform/imx8mp-verdin/inc/plat/platform.h

```diff
@@ -0,0 +1,11 @@
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Bao Project and Contributors. All rights reserved.
+ */
+
+#ifndef __PLAT_PLATFORM_H__
+#define __PLAT_PLATFORM_H__
```

Comment:
```suggestion
#ifndef PLAT_PLATFORM_H
#define PLAT_PLATFORM_H
```

Let's try to not introduce more identifiers starting with ``__``  which reserved by the C standard.